src/Features/CSharp/Portable/Completion/Providers/DefaultArgumentProvider.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
+{
+    [ExportArgumentProvider(nameof(DefaultArgumentProvider), LanguageNames.CSharp)]
+    [ExtensionOrder(After = nameof(FirstBuiltInArgumentProvider))]
+    [Shared]
+    internal sealed class DefaultArgumentProvider : AbstractDefaultArgumentProvider
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public DefaultArgumentProvider()
+        {
+        }
+
+        public override Task ProvideArgumentAsync(ArgumentContext context)
+        {
+            if (context.PreviousValue is { })
```

Comment:
can we use `is not null`?

Reply:
No objection, but left it like this to expedite reviews on #49916.